src/shared/services/api/api.ts

```diff
@@ -11,7 +14,7 @@ const getAccessToken = () => getState()?.user?.accessToken
 export const fetchWithoutToken = <T = any>(uri: string): Promise<T> =>
   fetch(uri).then((response) => response.json())
 
-const handleErrors = (response: Response, reloadOnUnauthorized: boolean) => {
+const handleErrors = (response: Response, reloadOnUnauthorized: boolean = false) => {
```

Comment:
When providing a default value it is not needed to specify the type as it can be inferred.

```suggestion
const handleErrors = (response: Response, reloadOnUnauthorized = false) => {
```